install-opencode.sh

```
@@ -0,0 +1,348 @@
+
+echo ""
+echo -e "${GREEN}Target directory: ${TARGET_DIR}${NC}"
+echo ""
+
+# Phase 1: Collection
+echo -e "${BLUE}Phase 1: Scanning files...${NC}"
+echo ""
+
+# Create temp file to store file metadata
+TEMP_FILE=$(mktemp)
+trap "rm -f $TEMP_FILE" EXIT
```

Shellcheck (0.11.0)

[warning] Use single quotes, otherwise this expands now rather than when signalled.

(SC2064)